annotation/src/test/java/org/cbioportal/genmone_nexus/annotation/util/MutationAssessorTest.java

```diff
@@ -0,0 +1,98 @@
+package org.cbioportal.genmone_nexus.annotation.util;
+
+import org.cbioportal.genome_nexus.annotation.service.internal.MutationAssessorService;
+import org.cbioportal.genome_nexus.annotation.domain.MutationAssessor;
+import org.cbioportal.genome_nexus.annotation.util.Transformer;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.logging.*;
+import java.net.URL;
+
+import static org.junit.Assert.*;
+
+public class MutationAssessorTest
+{
+    /* for debugging
+    private static Logger log = Logger.getLogger(String.valueOf(MutationAssessorTest.class));
+    */
+
+    @Test
+    public void testVariants() throws IOException {
+
+        MutationAssessorService service = new MutationAssessorService();
+        service.setMutationAssessorURL("http://mutationassessor.org/r3/?cm=var&var=");
```

Comment:
@onursumer Yeah I told @athum to not worry about that for now. I can help address this issue in a separate PR.